src/container.rs

```diff
@@ -521,17 +521,10 @@ impl Container {
         program: &str,
         argv: &[&str],
     ) -> super::Result<i32> {
-        let argv_ptr = if argv.is_empty() {
-            null()
-        } else {
-            let mut argv: Vec<*const i8> = argv.iter().map(|e| to_cstr(*e)).collect();
-
-            argv.push(null());
-
-            argv.as_ptr()
-        };
+        let mut argv_: Vec<*const i8> = argv.iter().map(|e| to_cstr(*e)).collect();
```

Comment:
Can you find a better name than `argv_`? It’s possible to reuse `argv`.

Reply:
Thanks, vector renamed.